tests/foreman/ui/test_templatesync.py

```diff
@@ -77,11 +77,11 @@ def test_positive_import_templates(session, templates_org, templates_loc):
         )
         assert import_title == f'Import from {FOREMAN_TEMPLATES_COMMUNITY_URL}'
         imported_template = f'{prefix_name} {import_template}'
+        assert session.provisioningtemplate.is_locked(imported_template)
         pt = session.provisioningtemplate.read(imported_template)
         assert pt['template']['name'] == imported_template
         assert pt['template']['default'] is False
         assert pt['type']['snippet'] is False
-        assert pt['template']['locked'] is True
```

Comment:
Isn't it returning/reading the `locked` parameter data now ?

Reply:
Not sure I fully understand the question but `pt['template']` has no attribute `locked` (https://github.com/SatelliteQE/airgun/blob/master/airgun/views/provisioning_template.py#L61) so I had to implement a different way to check whether the template is locked, hence the new method `is_locked` in the mentioned airgun PR.